### ISSUE TITLE
[@typescript-eslint/member-ordering] wrong/unclear behavior or documentation

### ISSUE THREAD

**Issue by DaSchTour** [typescript-eslint/typescript-eslint#4285]
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.
- [x] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/main/docs/linting/TROUBLESHOOTING.md) and my problem is not listed.

**Repro**

```JSON5
{
  "rules": {
    "@typescript-eslint/member-ordering": [
      "error",
      {
        "default": {
          "memberTypes": [
            // Fields
            "decorated-field",
            "field",

            // Constructors
            "constructor",

            // Methods
            "decorated-method",
            "method"
          ]
        }
      }
    ]
  }
}
```

```TS
class Foo {
  public methodOne() {}
  private methodTwo() {}
  public methodThree() {}
}
```

**Expected Result**

methodThree() should be marked as beeing in the wrong order. 

As from the documentation `method` is equivalent to `["public-static-method", "protected-static-method", "private-static-method", "public-instance-method", "protected-instance-method", "private-instance-method", "public-abstract-method", "protected-abstract-method", "private-abstract-method"]`.

When changing 'method' to 'public-method', 'private-method' in works as expected. 

**Actual Result**

no lint warning

**Additional Info**

No

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `5.6.0` |
| `@typescript-eslint/parser`        | `5.6.0` |
| `TypeScript`                       | `4.4.4` |
| `ESLint`                           | `7.32.0` |
| `node`                             | `14.17.6` |

**Comment by bradzacher** [comment]
the default `order` for any config is `as-written`.
if you set your `order` to `alphabetically`, it will do what you want.

**Comment by DaSchTour** [comment]
@bradzacher so I have to add everything explicitly? Is there any possibility to get the default order public protected  private for any group without explicitly adding them?

**Comment by bradzacher** [comment]
The idea is that `method` is contextless so that you can sort methods without caring about their specifiers.

There's little need for creating a standard ordering of specifiers because it's such a stylistic thing and it's 2 extra lines per type to explicitly specify the ordering you would like.
